vnet/tbf.go

```diff
@@ -0,0 +1,93 @@
+package vnet
+
+import (
+	"context"
+	"time"
+)
+
+const (
+	// Bit is a single bit
+	Bit = 1
+	// KBit is a kilobit
+	KBit = 1000 * Bit
+	// MBit is a Megabit
+	MBit = 1000 * KBit
+)
+
+// TokenBucketFilter implements a token bucket rate limit algorithm.
+type TokenBucketFilter struct {
+	NIC
+	bucket int
```

Comment:
Should this be renamed to `bucketSize`?

Reply:
I think `bucketSize` could be misleading, because it is the number of tokens currently in the bucket.